metricbeat/module/kubernetes/util/prometheus.go

```diff
@@ -0,0 +1,48 @@
+package util
+
+import (
+	"fmt"
+
+	"github.com/elastic/beats/metricbeat/helper"
+
+	dto "github.com/prometheus/client_model/go"
+	"github.com/prometheus/common/expfmt"
+)
+
+func GetFamilies(http *helper.HTTP) ([]*dto.MetricFamily, error) {
```

Comment:
Should this be a `prometheus` helper under the `helper` namespace? Or can you use the `prometheus.collector` code here?